src/data/fin.lean

```diff
@@ -907,6 +912,17 @@ lemma succ_above_below (p : fin (n + 1)) (i : fin n) (h : i.cast_succ < p) :
   p.succ_above i = i.cast_succ :=
 by { rw [succ_above], exact if_pos h }
 
+@[simp] lemma zero_succ_above_eq_zero {a : fin (n + 2)} (ha : a ≠ 0) : 0 = a.succ_above 0 :=
```

Comment:
This seems like a bad simp-lemma, since the RHS of the equation is more complex than the LHS. So I suggest
```suggestion
@[simp] lemma zero_succ_above_eq_zero {a : fin (n + 2)} (ha : a ≠ 0) : a.succ_above 0 = 0 :=
```

Reply:
silly mistake, thanks!